cargo-workspaces/src/publish.rs

```diff
@@ -166,7 +166,11 @@ impl Publish {
             }
         }
 
-        info!("success", "ok");
+        if command_success {
```

Comment:
```suggestion
        if self.dry_run {
```

And let's just ignore the failure scenario because the user can already see the failures above. Adding a text here felt like noise to me.